model/adjuster/clockskew.go

```diff
@@ -52,12 +53,14 @@ func ClockSkew() Adjuster {
 const (
 	warningDuplicateSpanID       = "duplicate span IDs; skipping clock skew adjustment"
 	warningFormatInvalidParentID = "invalid parent span IDs=%s; skipping clock skew adjustment"
+	warningMaxSkewAdjustExceeded = "max skew adjust of %v exceeded.  not applying calculated adjustment of %v"
 )
 
 type clockSkewAdjuster struct {
-	trace *model.Trace
-	spans map[model.SpanID]*node
-	roots map[model.SpanID]*node
+	trace         *model.Trace
+	spans         map[model.SpanID]*node
+	roots         map[model.SpanID]*node
+	maxSkewAdjust time.Duration
```

Comment:
```suggestion
	maxDelta time.Duration
```
(we already know it's for skew adjustment from the type name)